boards/sipeed/longan_nano/longan_nano_gd32vf103.yaml

```diff
@@ -1,4 +1,4 @@
-identifier: longan_nano
+identifier: longan_nano/gd32vf103
```

Comment:
(optional) base board does not need the soc in the name or identifier

Reply:
I guess that will save me from updating a few twister files right?